com.unity.ml-agents/Runtime/Policies/BarracudaPolicy.cs

```diff
@@ -64,14 +63,8 @@ public void RequestDecision(AgentInfo info, List<ISensor> sensors)
         public ref readonly ActionBuffers DecideAction()
         {
             m_ModelRunner?.DecideBatch();
-            var actions = m_ModelRunner?.GetAction(m_AgentId);
-            if (m_SpaceType == SpaceType.Continuous)
-            {
-                m_LastActionBuffer = new ActionBuffers(actions, Array.Empty<int>());
-                return ref m_LastActionBuffer;
-            }
-
-            m_LastActionBuffer = ActionBuffers.FromDiscreteActions(actions);
+            var actionbuffer = m_ModelRunner?.GetAction(m_AgentId);
```

Comment:
Not sure how `?.` works with structs. Maybe best to write this as something like
```
if(m_ModelRunner == null) {
  return ActionBuffers.Empty;
}
m_ModelRunner.DecideBatch();
return m_ModelRunner.GetAction(m_AgentId);
```